src/app/_components/CheckBox/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as CheckBox } from './CheckBox';
+export { checkBoxVariants } from './styles';
```

Comment:
🟢 do you need to expose styles? 🤔

Reply:
it's exposed to allow using different variants for the checkbox, in case, new variants are added to the styles.

Reply:
yes, i mean since this is `index` file, it kinda tells that everything in `index` should be used from outside, and i can't imagine a case for that

Reply:
I guess you're right. I've removed the variants from the index file